dependencies.gradle

```diff
@@ -9,7 +9,7 @@ ext {
     mockWebServerVersion = '3.8.1'
     robolectricVersion = '3.6.1'
     hamcrestVersion = '1.3'
-    stellarSDKVersion = '3b05787858d847be59f3dc6333ed44887ce5cc1b'
+    stellarSDKVersion = 'bfc53061e14ff6d2b83b44f5adba5ce1d1a91466'
```

Comment:
Can't we use versions?

Reply:
I did in the past, but I don't want to maintain versions for that, as this is a internal library (meaning we are not targeting it for outside use currently), I would prefer actually to have `android-stellar-sdk` as submoudle, but didn't have the chance to do it by now